Text/AbbyyOCR/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
So far, we haven't had a different license & copyright per skill. The top-level license is MIT anyways.